search-enterprise/latest/admin/index.md

```diff
@@ -496,3 +496,89 @@ The main use case to re-index only content or path is a fully metadata indexed r
+| elasticsearch.healthcheck.date.minRange | The minimum healthcheck database node update date. A node must have a database update date that is greater than or equal to the specified value that is included in the healthcheck. The default is `2023-01-01T23:59:00Z`. |
+| elasticsearch.healthcheck.date.maxRange | The maximum healthcheck database node update date. A node must have a database update date that is less than or equal to the specified value that is included in the healthcheck. The default is `2023-01-07T23:59:00Z`. |
+| elasticsearch.healthcheck.batchSize | The number of nodes aggregated from the database to be checked in Search Enterprise in one go. The property cannot be greater than `10000`. The default is `10000` |
+| elasticsearch.healthcheck.confidenceThresholdInMs | A threshold value in milliseconds that can be used to help you avoid false positives. It can be used in cases where the timestamp difference between the database and Search Enterprise is below the threshold. An example of this is when you are managing out of order events, or in circumstances where you have expected delays in indexing. The default is `1000`. |
+| elasticsearch.healthcheck.pollingRatio | The distance between nodes to check. For example, if you set this property  to `10`, then every 10th node within the configured range is checked. The default is `1`. |
+| elasticsearch.healthcheck.timeoutInHours | The number of hours the healthcheck is allowed to run. If the time out is exceeded, the healthcheck is stopped and only the results found until the health check is stopped will be displayed. The default is `1`. |
+| elasticsearch.healthcheck.startTime | The scheduled time to automatically start the health check. The default is `2030-12-30T23:59:00Z`. |
+| elasticsearch.healthcheck.intervalPeriod | The period of time the health check should wait between repeated executions after the first scheduled execution. The possible values are: `Month`, `Week`, `Day`, `Hour`, `Minute`, or `Second`. The default is `Week`. |
+| elasticsearch.healthcheck.intervalCount | Sets how many periods should be waited between each scheduled execution. The default is `1`. |
+| elasticsearch.healthcheck.nodeAspectsToExclude | A comma-separated list of node aspects. Nodes with any of the aspects from the specified list will be excluded from the health check. The default is `sys:hidden`. |
+
+> **Note:** You can only set one range format, either ID or Date. This means two properties for one of the range formats must always be empty.
+
+### Run Healthcheck Job
+
+Use the JConsole to run the Healthcheck.
+
+  **Note:** The MBean `Alfresco:Name=ElasticsearchHealthcheck` is exposed and allows you to manage the Healthcheck.
+
+1. Open a command prompt and `cd` to your JDK installation directory.
+
+2. Open the Java Monitoring & management Console window by entering: `jconsole`.
```

Comment:
I'm not sure what the correct capitalization is, but can we use initial capitals?
```suggestion
2. Open the Java Monitoring & Management Console window by entering: `jconsole`.
```